docs/UserGuide.md

```diff
@@ -439,6 +439,12 @@ Furthermore, certain edits can cause the mediCLI to behave in unexpected ways (e
 
    **Workaround**: Users can manually extract data from the JSON data file used by MediCLI and convert it to the desired format using third-party tools or scripts. Alternatively, they can explore custom export solutions or request this feature from the developers.
 
+5. **Issue**: When the name entered into the system is too lengthy, MedicCLI truncates the name and adds ellipses.
+
+   **Impact**: Users may face challenges reading or finding long names when reading from MediCLI, especially on smaller displays.
+
+   **Workaround**: Users can reduce the amount of characters typed into the name field or search for longer names based on the first few characters shown instead of the entire name .
+
```

Comment:
suggestion 

```
Issue: When names entered into the system exceed a certain length, MedicCLI truncates them and adds ellipses.

Impact: Users may encounter difficulties in reading or locating long names within MedicCLI, particularly on smaller displays.

Workaround: Users can utilise abbreviations or aliases for long names when entering them into the system. However, it's essential to ensure that these abbreviations or aliases are clear and consistent to avoid confusion or errors in data management.
```